Meraki.Api/Data/FirmwareProducts.cs

```diff
@@ -63,7 +63,7 @@ public class FirmwareProducts
 	public FirmwareUpgradeProduct Wireless { get; set; } = new();
 
 	/// <summary>
-	/// cloudGateway
+	/// cloudGateway - Deprecated from v1.38
```

Comment:
There is an actual [Deprecated] attribute I believe that you could apply to this, then when you go to use it in some code the IDE will let you know it's deprecated. Could you check this out and add as appropriate.